libraries/postgres/test/database_test.js

```diff
@@ -140,6 +140,8 @@ helper.dbSuite(path.basename(__filename), function() {
         await client.query('grant trigger on foo to test_service1');
         await client.query('grant select on foo to test_service2');
         await client.query('grant select on bar to test_service2');
+        // a column grant should get revoked, too
+        await client.query('grant select (x) on bar to test_service1');
```

Comment:
Oh, I need to re-add a test for this.  It's adding a *column* grant, instead of a table grant.  The intent is to make sure that the permissions-update code notices and corrects this situation, just in case someone maliciously or accidentally adds a column grant to the DB.

Reply:
I think the `(x)` confused me. Are we granting to a column with name `(x)`?

Reply:
Yes, exactly.

Reply:
I [used a better column name](8bb363b6501b3a77e667024b22d4d803b22b7a49)